### PR TITLE
new German brands

### PR DESCRIPTION
**Pull request by Contrib1043**

### CODE REVIEW

**Comment by bhousel** [comment]
Thanks @Contrib1043 - just wanted to remind you that you're listed as a maintainer on this project and can merge in any brands directly to `main` if you want to.. You don't need to open a pull request for things that you know well  👍   

Also double check the syntax, since I see some warnings in the diff about missing commas.

File: data/brands/shop/furniture.json [hunk 1]
```diff
@@ -819,6 +842,18 @@
...
     },
+    {
+      "displayName": "Plana Küchenland",
+      "locationSet": {"include": ["de"]},
+      "tags": {
+        "brand": "Plana Küchenland",
+        "brand:wikidata": "Q1576634",
+        "brand:wikipedia": "de:Plana Küchenland",
+        "name": "Plana Küchenland",
+        "short_name": "Plana"
```
Comment (tas50) [inline data/brands/shop/furniture.json:853]:
```suggestion
        "short_name": "Plana",
```

File: data/brands/shop/furniture.json [hunk 2]
```diff
@@ -819,6 +842,18 @@
...
+        "shop": "furniture"
```
Comment (tas50) [inline data/brands/shop/furniture.json:854]:
```suggestion
        "shop": "furniture",
```

File: data/brands/shop/shoes.json [hunk 3]
```diff
@@ -71,6 +71,17 @@
         "shop": "shoes"
       }
     },
+    {
+      "displayName": "anika schuh",
+      "locationSet": {"include": ["de"]},
+      "tags": {
+        "brand": "anika schuh",
+        "brand:wikidata": "Q1815747",
+        "brand:wikipedia": "de:Leiser Handelsgesellschaft"
```
Comment (tas50) [inline data/brands/shop/shoes.json:80]:
```suggestion
        "brand:wikipedia": "de:Leiser Handelsgesellschaft",
```

File: data/brands/shop/shoes.json [hunk 4]
```diff
@@ -786,6 +797,17 @@
         "shop": "shoes"
       }
     },
+    {
+      "displayName": "Leiser",
+      "locationSet": {"include": ["de"]},
+      "tags": {
+        "brand": "Leiser",
+        "brand:wikidata": "Q1815747",
+        "brand:wikipedia": "de:Leiser Handelsgesellschaft"
```
Comment (tas50) [inline data/brands/shop/shoes.json:806]:
```suggestion
        "brand:wikipedia": "de:Leiser Handelsgesellschaft",
```

**Pull request merged**